src/libOpenImageIO/imageoutput.cpp

```diff
@@ -80,6 +86,13 @@ ImageOutput::operator delete(void* ptr)
 ImageOutput::ImageOutput()
     : m_impl(new Impl, impl_deleter)
 {
+    // Ensure that if this ImageOutput has the same address as a previous one
```

Comment:
Did you mean to leave this in? The comment refers to the address, but is using the id.

Reply:
I did not. Fixing.